pyquil/_parser/grammar.lark

```diff
@@ -60,7 +60,7 @@ def_circuit : "DEFCIRCUIT" name [ variables ] [ qubit_designators ] ":" indented
 //             | "DEFCIRCUIT" name [ variables ] qubit_designators ":" indented_instrs -> def_circuit_qubits
 
 def_frame : "DEFFRAME" frame ( ":" frame_spec+ )?
-frame_spec : _NEWLINE_TAB frame_attr ":" (expression | string )
+frame_spec : _NEWLINE_TAB frame_attr ":" ( expression | string )
```

Comment:
Ah, so it _already_ was `string`. I must've been looking at the wrong/old branch.